RIS.js

```diff
@@ -405,6 +404,7 @@ var degenerateImportFieldMap = {
 	CA: "unsupported/Caption",
 	CR: "rights",
 	CT: "title",
+	CY: "place",
```

Comment:
Actually, this doesn't seem right, either. `CY` is already included for place, just not for conference papers see https://github.com/zotero/translators/blob/72c97717037a1da8ff5310deaccc4e2a97571e0d/RIS.js#L287

Reply:
Not exporting is okay, but why should we not import it if it is there? (There are similar cases in the RIS translator already e.g. for NV.)

Example:
```ris
TY  - CONF
AU  - Rusch, Anja
AU  - Wille, Rudolf
ED  - Bock, Hans-Hermann
ED  - Polasek, Wolfgang
PY  - 1996
DA  - 1996//
TI  - Knowledge Spaces and Formal Concept Analysis
BT  - Data Analysis and Information Systems
SP  - 427
EP  - 436
PB  - Springer Berlin Heidelberg
CY  - Berlin, Heidelberg
AB  - J.-P. Doignon and J.-C. Falmagne have introduced a formal notion of knowledge space to develop a scientific approach to the assessment of knowledge. They define a knowledge space as a (finite) set Q of questions together with a collection K, of subsets of Q representing different knowledge states; in addition, they assume that K is closed under set unions. The theory of knowledge spaces can be effectively connected with formal concept analysis. The connection is established by the definition of a knowledge context as a triple (P, Q, I) where P is a finite set of persons, Q is a finite set of questions, and I is a binary relation between P and Q such that plq means: the person p cannot solve the question q. For each knowledge context there is a corresponding knowledge space which consists of the complements of all intents of this context; conversely, each knowledge space can be derived in this way. Via the described connection, methods of formal concept analysis can be successfully applied to the theory of knowledge spaces. In particular, the attribute exploration as a method of conceptual knowledge acquisition yields a new approach for building knowledge spaces.
SN  - 978-3-642-80098-6
ID  - 10.1007/978-3-642-80098-6_36
ER  - 
```

Alternatively, I can also make a fix only in Springer Link translator, but I don't see any reason not to import this RIS field for any source if it is there. Let me know if you disagree and I am missing something.

Reply:
I agree with import in the absence of `CI`, yes  -- I only checked quickly, but we do already have code that looks like it'd handle the same scenario: https://github.com/zotero/translators/blob/72c97717037a1da8ff5310deaccc4e2a97571e0d/RIS.js#L1036 
Might make sense to check if we can then take this out or alternatively use this version instead of the hard to understand degenerateImport fields? (but no strong preference either way; just don't want duplicate code)

Reply:
Okay, I will look into this more closely at a later point and also check the code you have found.

Reply:
Uff, that is a little complicated... The code you found is only considered for RIS from ProCite. When I try to take this out of this special part, then the type is not yet defined. But I cannot replace CY with C1 always, as this would destroy the cases for books as an example.

Thus, I will continue with the degenerateImport fields which I find easier to work with.